ee/WFLY-19062-[PREVIEW]-Support_for_Jakarta_MVC_2.1.adoc

```diff
@@ -0,0 +1,175 @@
+* Extension will advertise itself as having stability level 'preview'.
+* JBoss Modules modules associated with the feature will not be provisioned if the user has disabled provisioning of 'preview' stability features.
+
+=== Nice-to-Have Requirements
+// Requirements in this section do not have to be met to merge the proposed functionality.
+// Note: Nice-to-have requirements that don't end up being implemented as part of
+// the work covered by this proposal should be moved to the 'Future Work' section.
+
+
+=== Non-Requirements
+// Use this section to explicitly discuss things that readers might think are required
+// but which are not required.
+ * Inclusion of the extension/subsystem in any standard OOTB config, including docs/examples configs.
+ * Inclusion in any of the WildFly user boms.
+ * Quickstart.
+
+=== Future Work
+// Use this section to discuss requirements that are not addressed by this proposal
+// but which may be addressed in later proposals.
+* Subsequent promotion to 'community' stability.
+* Possible move into the 'wildfly-ee' feature pack.
+** This would reflect a 'promotion' in terms of WildFly's long term compatibility expectations for Jakarta MVC itself.
+* Add a quickstart
+** Requires promotion to 'community' or Quickstart project support for preview features.
+
+== Backwards Compatibility
+
+// Does this enhancement affect backwards compatibility with previously released
+// versions of WildFly?
+// Can the identified incompatibility be avoided?
+
+=== Default Configuration
+
+No incompatibility.
+
+=== Importing Existing Configuration
+
+No incompatibility.
+
+=== Deployments
+
+No incompatibility.
+
+=== Interoperability
+
+No incompatibility.
+
+//== Implementation Plan
+////
+Delete if not needed. The intent is if you have a complex feature which can 
+not be delivered all in one go to suggest the strategy. If your feature falls 
+into this category, please mention the Release Coordinators on the pull 
+request so they are aware.
+////
+Use WildFly Extras to house the source code. Incorporate into the 'wildfly' and 'wildfly-preview' feature packs in a manner similar to how we incorporate content from WildFly Core into the 'wildfly-ee' feature pack.
+
+== Security Considerations
+
+////
+Identification if any security implications that may need to be considered with this feature
+or a confirmation that there are no security implications to consider.
+////
+None.
+
+== Test Plan
+
+ * Standard subsystem tests (parsing and marshalling; Stage.MODEL execution)
+ ** Done in the mvc-krazo-feature-pack project.
+ * Pass the Jakarta MVC TCK
+ ** TCK runner should support passing in the GAV of the feature pack to provision the 
+    server, allowing execution against both the wildfly-preview feature pack and
+	the mvc-krazo-specific feature pack
+ * Smoke test in the testsuite/integration/manualmode module
+ ** In the future to be moved to testsuite/integration/basic (perhaps for WildFly 32)
```

Comment:
Note I meant to say "perhaps for WildFly 32 Final". It's doable, but probably not for 32 Beta as it's not urgent and I have other tasks. It requires some test fixture stuff Kabir did that only came into full WF this week.